packages/proxy/src/proxy.ts

```diff
@@ -490,6 +527,10 @@ export async function proxyV1({
     }
     setHeader(LEGACY_CACHED_HEADER, "false"); // We're moving to x-bt-cached
     setHeader(CACHED_HEADER, "MISS");
+    if (useCache) {
```

Comment:
do we need to check this? if we're in this path, then we necessarily must have missed the cache right?

Reply:
oh jk, i see